src/api/reservations/reservation.service.ts

```diff
@@ -15,18 +15,12 @@ export const getByIdReservation = (id: string) => {
 };
 
 export const createReservation = (input: any) => {
-  const {
-    date_hour,
-    restaurantsId_restaurant,
-    restaurant_venuesId_restaurant_venue,
-    usersUser_id,
-  } = input;
   return prisma.reservations.create({
     data: {
-      date_hour,
-      restaurantsId_restaurant,
-      restaurant_venuesId_restaurant_venue,
-      usersUser_id,
+      date_hour: input.date,
+      restaurantsId_restaurant: input.id_restaurant,
+      restaurant_venuesId_restaurant_venue: input.id_venue,
+      usersUser_id: input.user,
```

Comment:
destructuring could be an enhancement.

Reply:
I tried to write it like this but doesn't work
const {
  date_hour: date,
  restaurantsId_restaurant: id_restaurant,
  restaurant_venuesId_restaurant_venue: id_venue,
  usersUser_id: user
} = input;

Reply:
ok, no problem, thx 4 updating.